### ISSUE TITLE
Model Loading is now broken. 

### ISSUE THREAD

**Issue by kevinkmcguigan** [NSCC-COGS/Aestheta#102]
@donuty-party 
It seems that you're addition to to the model loading aspect of the script no longer functions. I've manually rolled back and removed the import of the new model.py code so that other features can be used and people can call things in colab etc. We may need to reassess how this model system works. We can either roll back to the original functionality or setup a hierarchy of packages where core.py and model.py are both sub-packages. 

I'd like to get back to our original state for now. We may need to discuss this before going forward. Currently we cannot load models. 

```
Traceback (most recent call last):
  File "c:\_dev\Aestheta\Library\core.py", line 22, in <module>
    from .model import Model # currently not working
ImportError: attempted relative import with no known parent package
```

**Comment by kevinkmcguigan** [comment]
Perhaps we can use your branch to try and implement the sub-package structure while I roll back to the original setup in our main branch?

**Comment by donuty-party** [comment]
@kevinkmcguigan, this is because of the relative import (`from .module import Class`). You need to run the script as `python -m Library.core` instead of `python Library/core.py`. I want to break code out of core.py to prevent it from getting overwhelming. Do you of a way to do that without requiring relative imports?

**Comment by donuty-party** [comment]
@kevinkmcguigan Oh, never mind. Absolute imports work fine and don't require us to change how we run the script. I have fixed this in my branch.

**Comment by donuty-party** [comment]
Fixed in 02a5267. This restores the original loadModel() and saveModel() functions, but in so doing, removes the model load by name functionality. I have moved that to the 82-load-model-by-name branch and reopened issue #82.

**Comment by kevinkmcguigan** [comment]
Thanks for resolving this - we'll sort things out in your branch and move them over to the main.
